src/http/validations/device.js

```diff
@@ -34,4 +34,19 @@ const updateDevice = (req, res, next) => {
   return next();
 };
 
-export default { getSingleDevice, updateDevice };
+const verifyDeviceLabel = async (req, res, next) => {
+  const { label } = req.body;
+  const { user, userType } = res.locals;
+  const deviceMatch = {
+    label,
```

Comment:
Convert this label to lowercase before using it for the DB query, this is so that we don't have the same label with just different letter casing.

Reply:
Fixed